src/plat/zynqmp/config.cmake

```diff
@@ -77,6 +78,9 @@ if(KernelPlatformZynqmp)
         CLK_MAGIC 1374389535llu
         CLK_SHIFT 37u
         KERNEL_WCET 10u
+        SMMU drivers/smmu/smmuv2.h
+        MAX_SID 32767
```

Comment:
Can we use `SMMU_SID_CNODE_SLOT_BITS` and `SMMU_CB_CNODE_SLOT_BITS` here
and then have in the generc code 
```
#define SMMU_MAX_SID BIT(SMMU_SID_CNODE_SLOT_BITS)
#define SMMU_MAX_CI BIT(SMMU_CB_CNODE_SLOT_BITS)
```